katas/chicken-sexing/jeanpeare/chickenSexing.js

```diff
@@ -1,3 +1,20 @@
 export function correctness(bobsDecisions, expertDecisions) {
-
+    var i = bobsDecisions.length;                                   // Get the lenght of the options array
+    var j = 0;                                                      // Declare the variable for looping
+    var points = 0;                                                 // Declare the variable for points
+    while (j<i) {
+        if (bobsDecisions[j] == expertDecisions[j]) {               // Validation of the condition with same opinions
+        points = points + 1;                                        // + 1 point
```

Comment:
Please fix indentation so alignment makes sense with the nesting inside the `if` block